purchases/src/test/java/com/revenuecat/purchases/paywalls/PaywallDataTest.kt

```diff
@@ -0,0 +1,51 @@
+package com.revenuecat.purchases.paywalls
+
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import java.io.File
+import java.net.URL
+import java.util.*
+
+class PaywallDataTest {
+
+    @Test
+    fun `test PaywallData properties`() {
+        val json = File("src/test/resources/paywalldata-sample1.json").readText()
```

Comment:
Is there a way to hardcode this path?

Reply:
Looks like this is the way? https://www.baeldung.com/junit-src-test-resources-directory-path#classloader

Reply:
Oh interesting let me check